src/nora/components/Example/Example.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react'
 import PropTypes from 'prop-types'
+import React from 'react'
```

Comment:
My setup is reordering these

Reply:
Interesting. We should see if we should codify this in a linting rule. Otherwise, down the road the git blame and PR diffs might get confusing. 

I’m in favor of grouping external and internal imports, but don’t have a particular opinion of ordering within that

Reply:
Good point Steph. So, I figured it out. It's `source.organizeImports` 

```json
"editor.codeActionsOnSave": {
  "source.organizeImports": true,
  "source.fixAll": true
}
```

I think I'll just remove it and place this back. I feel like it might be more idiomatic to have React at the top...I'll push up another commit once I do.

Reply:
Awesome thanks for digging into it Rob!